api/shareable.py

```diff
@@ -0,0 +1,55 @@
+from bson.objectid import ObjectId
+
+from pydantic import BaseModel
+from pymongo import MongoClient
+
+from loguru import logger
+
+
+class PlanData(BaseModel):
+    json_string: str
+    validation_levels: list[int]
```

Comment:
Maybe let's call this "validator_overrides" or something like that.  This name isn't very obvious to me.

Reply:
made the change, thanks for pointing that out.